lib/util/utils.js

```diff
@@ -511,7 +511,7 @@ class Utils {
   }
 
   static relativeUrl(url) {
-    return !(url.startsWith('http://') || url.startsWith('https://'));
+    return !(url.indexOf('://') > 0 || url.indexOf('//') === 0);
```

Comment:
.startsWith is a bit more elegant and what would be the use case for `url.indexOf('//') === 0`? What kind of uris would this cover?

Reply:
`//` is Protocol-relative URL,  e.g in a HTML `src="//ajax.googleapis.com` resolves to either `http://` or `https://` depending on current protocol.
Probably only useful with URLs inside a HTML/JS, and not much for test case navigation, but I am not sure if `relativeUrl` will be used against URL extracted from inside webpages.
Will remove this condition if you think its redundant.

Reply:
`.includes()` is probably what you need here. The protocol relative urls won't work in this case.

Reply:
Updated to use `.includes()` on `://` and removed protocol relative urls